sources/packages/backend/apps/api/src/route-controllers/report/report.aest.controller.ts

```diff
@@ -66,21 +64,8 @@ export class ReportAESTController extends BaseController {
     @Body() payload: ReportsFilterAPIInDTO,
     @Res() response: Response,
   ): Promise<void> {
-    const submissionResult = await this.formService.dryRunSubmission(
-      FormNames.ExportFinancialReports,
```

Comment:
Please refer to the previous comment. https://github.com/bcgov/SIMS/pull/1681/files#r1087210193

I believe we must revert this.